File: agent-driven-architecture-main/examples/typescript/test/gate/fixtures/compliant/C17/src/blocks/escalation/fold.ts
// COMPLIANT: the leaf at its OWN PINNED SITE — the arm of the Irreversible verb
// that earns it. This is the construction the roster in test/gate/c17.ts names,
// and it must pass untouched or the check is a rule nobody can satisfy.
import type { PageOncall } from "./contract";

export const page = (at: number, ticket: string): PageOncall => ({
  kind: "PageOncall",
  at,
  effectClass: "Irreversible",
  ticket,
});

File: agent-driven-architecture-main/examples/kotlin/src/test/fixtures/konsist/compliant/C17/blocks/escalation/Fold.kt
// COMPLIANT: the leaf at its OWN PINNED SITE — the arm of the Irreversible verb that
// earns it. This is the construction IRREVERSIBLE_SITES names, and it must pass
// untouched or the check is a rule nobody can satisfy.

package adr.blocks.escalation

import adr.contract.EscalationEffect
import adr.spine.pure.TicketId
import adr.spine.pure.Timestamp

class EscalationArm {
    fun confirm(now: Timestamp, ticket: TicketId) = EscalationEffect.PageOncall(now, ticket)
}
